stratum/hal/lib/barefoot/bfrt_counter_manager.h

```diff
@@ -52,14 +54,20 @@ class BfrtCounterManager {
  private:
   // Private constructor, we can create the instance by using `CreateInstance`
   // function only.
-  explicit BfrtCounterManager(BfSdeInterface* bf_sde_interface_, int device);
+  explicit BfrtCounterManager(
+      BfSdeInterface* bf_sde_interface_,
+      BfrtP4RuntimeTranslator* bfrt_p4runtime_translator, int device);
```

Comment:
Can you move the sde interface second to last?
It's like in BfrtNode: <managers>, sde interface, device